test/serializer/basic/setTimeoutAndInterval.js

```diff
@@ -0,0 +1,9 @@
+// does contain:keep me
+(function() {
+    let f = function() { /* keep me */ };
```

Comment:
This test is a bit odd since a further optimization could be to not serialize these side-effects at all if the timer is cleared before the end of the tick. Since it will never execute.